src/app/modules/portal-shared/portal-shared.module.ts

```diff
@@ -20,14 +20,20 @@ import { DynamicFieldDirective } from './components/dynamic-field/dynamic-field.
 import { MultiSelectComponent } from './components/multi-select/multi-select.component';
 import { DynamicFormComponent } from './components/dynamic-form/dynamic-form.component';
 import { MatFormFieldModule } from '@angular/material/form-field';
-import { TranslateModule } from '@ngx-translate/core';
-
+import { AssesmentCardComponent } from './assesment-card/assesment-card.component';
+import {TranslateModule, TranslateLoader, TranslateService} from "@ngx-translate/core";
+import {TranslateHttpLoader} from '@ngx-translate/http-loader';
+import { HttpClientModule, HTTP_INTERCEPTORS, HttpClient } from '@angular/common/http';
```

Comment:
remove unwanted imports. Use interceptor in appmodule

Reply:
Removed